### DISCUSSION TITLE
Peppol reporting requirements for SubjectCN

### DISCUSSION THREAD

**Discussion by RichardVanMaaren** [OxalisCommunity/oxalis#594] [General]
As I understood as of mid 2023 OpenPeppol will require statistics reporting. Data is aggregated at some level for example, number of incoming/outgoing per country per SubjectCN of the AP's.

In general most of the data could be retrieved from SBDH. However the sending SubjectCN of the other party is not registered somewhere.

Just heads up that this will come and we probably need something to retrieve this information from the transmissions
![Capture peppol reporting](https://user-images.githubusercontent.com/58323199/172568292-677749f2-cb00-4f87-9bf1-1f4f8432e72b.JPG)
.

**Comment by aaron-kumar** [comment]
Sending and receiving Access Point Subject Common Name available as part of certificate exchanges in AS4 profile. So all required information is there. There is plan to implement (at least) Transaction Reporting in Oxalis as part of reporting BIS epic: https://github.com/OxalisCommunity/Oxalis-Public-Roadmap/issues/15
TBD: Whether this functionality will be available to paid members or general users.
